x/bank/keeper/keeper.go

```diff
@@ -6,11 +6,9 @@ import (
 	"cosmossdk.io/math"
 
 	errorsmod "cosmossdk.io/errors"
-	"cosmossdk.io/store/prefix"
 	storetypes "cosmossdk.io/store/types"
```

Comment:
in the future would we pass the kv store service and not depend on store here?

Reply:
ideally yes. for now we still depend on it because of KVStoreKey.